platforms/joystick/joystick_driver.go

```diff
@@ -31,6 +31,12 @@ const (
 
 	// Configuration for the Xbox One controller.
 	XboxOne = "xboxOne"
+
+	// Nvidia Shield TV Controller
+	Shield = "shield"
```

Comment:
seems this was pulled in when I was rebasing release -> dev because I accidentally started there.